rudder-core/src/main/resources/ldap/rudder.schema

```diff
@@ -484,7 +493,7 @@ objectclass ( RudderObjectClasses:23
   STRUCTURAL
   MUST ( directiveId $ techniqueVersion )
   MAY ( cn $ description $ longDescription $ isEnabled $ isSystem $
-        directivePriority $ directiveVariable  ) )
+        directivePriority $ directiveVariable $ policyMode ) )
```

Comment:
you need to update also the 099-1-rudder.ldif file in test/resources/ldap-data/schema